cve_bin_tool/cvedb.py

```diff
@@ -286,13 +286,12 @@ def init_database(self):
             versionStartIncluding TEXT,
             versionStartExcluding TEXT,
             versionEndIncluding TEXT,
-            versionEndExcluding TEXT
+            versionEndExcluding TEXT,
+            PRIMARY KEY(cve_number)
```

Comment:
My understanding was that this wouldn't work because sqlite requires the primary key to be unique, and cve_number is not unique in the range_table (it is in severity).  Is that not true for sqlite?

Reply:
Yes, you are right, the primary key needs to be unique in every SQL!  If the primary key isn't unique it will overwrite it when duplicate values passed. Currently, we are using indexes but we can create a composite key of several attributes and call it a primary key. I have updated the schema.